### DISCUSSION TITLE
What is an XP (Contribution Score)?

### DISCUSSION THREAD

**Discussion by olegcherev** [devactivity-app/devActivity-Community#17] [Q&A]
☝️

**Comment by damonexsppfx-alt** [comment]
Pls a proper explanation

**Reply by olegcherev** [reply]
Hi @damonexsppfx-alt — thanks for calling this out. Here is a proper explanation.

**XP (Contribution Score)** is how devActivity turns GitHub work into a single, comparable score. It is **not** a judgment of code quality and it does **not** read source code. It only uses repo metadata (commits, PRs, reviews, issues).

You earn XP from everyday work, for example:
- opening / merging **pull requests**
- doing **code reviews**
- opening, commenting on, and closing **issues**

Those points add up over time. They show up on the dashboard as charts (who contributed, which activity types, which weeks), feed **levels / achievements / leaderboards**, and give managers a fairer signal than “who talked the most in standup.”

Hover a bar on the XP widget to see the breakdown (PRs vs reviews vs issues). Click through for the full contribution list.

More detail:
- Overview: https://devactivity.com/help/reports/xp-contribution-score-overview/
- How to use the widget: https://devactivity.com/help/reports/how-to-use-xp-contribution-score-widget/
- FAQ: https://devactivity.com/help/faq/xp-contribution-score-faq/

If something still isn’t clear (how a specific event is scored, levels, challenges), tell us and we’ll add it here.
